plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,22 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewHistogram(
```

Comment:
Also I'm wondering if it won't be useful to split this data by prediciate computed (the same for priorities). @wojtek-t